R/ggplotly.R

```diff
@@ -934,22 +938,30 @@ gg2list <- function(p, width = NULL, height = NULL,
         strip <- make_strip_rect(xdom, ydom, theme, "top")
         gglayout$shapes <- c(gglayout$shapes, strip)
       }
-      row_txt <- paste(
-        plot$facet$params$labeller(
-          lay[names(plot$facet$params$rows)]
-        ), collapse = br()
-      )
-      if (is_blank(theme[["strip.text.y"]])) row_txt <- ""
-      if (inherits(plot$facet, "FacetGrid") && lay$COL != nCols) row_txt <- ""
-      if (robust_nchar(row_txt) > 0) {
-        row_lab <- make_label(
-          row_txt, x = max(xdom), y = mean(ydom),
-          el = theme[["strip.text.y"]] %||% theme[["strip.text"]],
-          xanchor = "left", yanchor = "middle"
-        )
-        gglayout$annotations <- c(gglayout$annotations, row_lab)
-        strip <- make_strip_rect(xdom, ydom, theme, "right")
-        gglayout$shapes <- c(gglayout$shapes, strip)
+      # Only FacetGrid has no cols
+      if (inherits(plot$facet, "FacetGrid")) {
+        row_txt <- if (!length(names(plot$facet$params$rows)) == 0) {
+          paste(
+            plot$facet$params$labeller(
+              lay[names(plot$facet$params$rows)]
+            ),
+            collapse = br()
+          )
+        } else {
+          ""
+        }
```

Comment:
Similar recommendation as https://github.com/plotly/plotly.R/pull/2373/files#r1742335996

Reply:
Thx for the review and the suggestions. Just committed both suggestions.